docs/decisiones/gestor-dependencias.md

```diff
@@ -0,0 +1,15 @@
+# Elección del gestor de dependencias
+
+La inmensa mayoría de los gestores de dependencias más usados para proyectos de JavaScript/TypeScript son compatibles entre sí y todos usan en general los mismos paquetes. Por tanto, y por experiencia propia, como criterio de selección del gestor de dependencias se va a tener en cuenta la velocidad a la que instala los paquetes y la facilidad de añadir/eliminar paquetes y gestionar sus veriones. Y como criterio secundario, pero también muy importante, es que se mantenga actualizado.
+
+Teniendo estos criterios en cuenta, como gestores de dependencias tenemos las siguientes:
+
+- __npm__: es el gestor de paquetes por defecto de Node.js. Es el gestor de paquetes más utilizado sobre todo en proyectos de JavaScript. Permite definir la versión específica que se necesita de un determinado paquete para que funcione el sistema y pueden gestionarse (actualizar, borrar, instalar,...) los paquetes desde la línea de comandos. Usa un fichero `package.json` donde se le indica la ruta de las dependencias instaladas dentro de la carpeta `node_modules`. También nos permite diferenciar entre dependencias de producción y dependencias de desarrollo
```

Comment:
Todo este párrafo, ¿de qué sirve para elegir uno u otro? ¿Que se instale en una carpeta u otra lo diferencia de los otros? ¿Lo hace mejor o peor?
Dicho de otro modo, os agradecería que pusiérais en un PR lo necesario para evaluar. Esto no es un TFG, no tenéis que meter literatura para llegar a un número de páginas.

Reply:
Si que es verdad que me ando por las ramas, lo cambio en el siguiente commit.